cr.sh

```diff
@@ -177,6 +177,17 @@ lookup_latest_tag() {
     fi
 }
 
+filter_changed_charts() {
```

Comment:
The function generally filters charts and has no notion of changed charts. Please change the name.
```suggestion
filter_charts() {
```